File: Java/arr1.java
public class arr1{
    public static void main(String[] args){
        int[] arr = { 10,20,30,40,50};
        int sum = 0; 
        float average;
        for(int i = 0; i<arr.length; i++){
            sum = sum+arr[i];
        }
        average = sum/5.0f;

        System.out.println("the sum is: " +sum);
    System.out.println("the average is : " +average);
    

    }
    

}

File: Java_Practice/C/arr1.c
#include<stdio.h>
int main()
{
    int arr[5] = {10,20,30,50,40};
    int sum = 0;
    float average;
    for(int i = 0; i<5; i++)
    {
        sum += arr[i];
    }
    average = sum/5.0;

    printf("the sum of :%d\n",sum);
    printf("the average of :%.2f\n",average);
    return 0;

}